src/index.html

```diff
@@ -22,6 +22,404 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="moyo_catalog"></div>
+    <header class="header">
+      <a
+        href="#"
+        class="moyo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
```

Comment:
The anchor tag has an href attribute set to '#', which might not be functional. Ensure that this is intentional or replace it with a valid URL if needed.